.changeset/orange-flowers-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ckb-lumos/common-scripts": patch
+---
+
+enable ACP to work with cells with more than 16 bytes of data
```

Comment:
enable ACP to work with sUDT cells with more than 16 bytes of data

Reply:
updated